L1Trigger/ME0Trigger/plugins/ME0TriggerProducer.cc

```diff
@@ -0,0 +1,47 @@
+#include "L1Trigger/ME0Trigger/plugins/ME0TriggerProducer.h"
+#include "L1Trigger/ME0Trigger/src/ME0TriggerBuilder.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DataFormats/GEMDigi/interface/ME0TriggerDigiCollection.h"
+#include "Geometry/Records/interface/MuonGeometryRecord.h"
+#include "Geometry/GEMGeometry/interface/ME0Geometry.h"
+
+ME0TriggerProducer::ME0TriggerProducer(const edm::ParameterSet& conf) 
+{
+  me0PadDigiProducer_ = conf.getParameter<edm::InputTag>("ME0PadDigiProducer");
+
+  trigBuilder_.reset( new ME0TriggerBuilder(conf) ); // pass on the conf
```

Comment:
Stream and global modules support multithreading; one modules are the worst type (except for legacy modules which shouldn't be used anymore). https://twiki.cern.ch/twiki/bin/view/CMSPublic/FWMultithreadedFrameworkModuleTypes

Global is better than stream, and it's possible for this producer if you follow what I outlined.

(Definitely do not follow the example of `CSCTriggerGeometry`, because it's completely thread-unsafe and will be removed whenever someone gets around to it.)